### DISCUSSION TITLE
Core Working Group Meeting Request | Selection of New Version Features

### DISCUSSION THREAD

**Discussion by ravi-prakash-v** [beckn/protocol-specifications#255] [Announcements]
As there are a significant number of updates to the core spec, we need to publish the next version of the Core Specification - Preferably version `1.0.0`. 

Requesting @beckn/core-working-group to suggest a meeting date, time and duration to conduct this call.

Preferred time are early mornings or late evenings IST as some of the members are working out of EST. 

My suggestion:

**Date**: Tuesday, 17th May, 2022
**Time**: 7:00 pm to 9:00 pm IST
**Duration** : 120 minutes

**Comment by pramodkvarma** [comment]
I am fine with the suggested date and time. Once confirmed, please send calendar request. Thanks.

**Comment by ravi-prakash-v** [comment]
@venkatramanm @sjthnrk @pkapustin kindly confirm the time. Thanks

**Comment by ravi-prakash-v** [comment]
Hi all, postponing the meeting request to next week as many of us have calendar conflicts. 

**Date:** Tuesday, 24th May, 2022
**Time:** 10:00 am to 12:00 pm
**Duration:** 120 minutes

@beckn/core-working-group please confirm. Thanks

**Reply by venkatramanm** [reply]
I am available only after 2:00pm any day. If this is an issue, please go ahead with others and will read up the minutes and get on same page.

**Reply by ravi-prakash-v** [reply]
@venkatramanm selection of features will require voting by all members of the working group, hence attendance is mandatory. But your timings are noted. I shall schedule it accordingly.

**Comment by ravi-prakash-v** [comment]
@pkapustin I received your message on Slack that you will not be able to attend due to some personal commitments. In that case, your vote on each decision will be counted as a "Yes". Requesting your confirmation on the same. Thanks.

**Reply by pkapustin** [reply]
@core-wg-admin I think it would be more correct to count votes of any members who are not present as "didn't vote" rather than "yes".

**Reply by ravi-prakash-v** [reply]
Good point @pkapustin . I think we need to add this in our governance mechanism. I don't think it is explicitly mentioned anywhere. But noted your point. Will count your vote as "Not Voted". Thanks for the suggestion.

**Comment by ravi-prakash-v** [comment]
@pramodkvarma @venkatramanm @sjthnrk can we do Friday, 3rd June 2022, 2:00 pm to 4:00 pm?
